lib/jsonapi_swagger_helpers/writeable.rb

```diff
@@ -1,21 +1,26 @@
+# frozen_string_literal: true
+
 module JsonapiSwaggerHelpers
   module Writeable
     def self.included(klass)
       klass.class_eval do
         attr_reader :node,
-          :controller,
-          :resource,
-          :description,
-          :tags
+                    :controller,
+                    :resource,
+                    :description,
+                    :tags,
+                    :singular
       end
     end
 
-    def initialize(node, controller, description: nil, tags: [])
+    def initialize(node, controller, description: nil, tags: [],
+                   singular: false)
```

Comment:
Can we keep this on the same line?

Reply:
Yeah no problem, I was just using rubocop with line limit of 80. It maybe a good idea to use some kind of static analyzer?

Reply:
I'd just kick em all down if we hit 80